storage/tables/index.md

```diff
@@ -96,13 +96,35 @@ not have any effect. You can mark a column as indexed in the table detail:
 
 ### Primary Key Deduplication
 When a primary key is defined on a column, the value of that column is guaranteed to be **unique** in that table.
-With a primary key defined on **multiple columns**, the combination of their values is unique. 
+As data are loaded into the table, only one of the rows with duplicate values is preserved.

```

Comment:
Doplnil bych, že není garantovaný pořadí, která z nich bude zachovaná. Možná bych celou tuhle sekci hodil do advanced? Začátečníky asi tohle úplně zajímat nemusí? Sorry, že vybíhám mimo scope původní úpravy! Můžem to nechat na jinej PR.

Reply:
je to tam hned zatim

Reply:
Myslíš 

> The order of rows in the imported file is not important and is not kept.

To ale není totéž.

Reply:
jakto?